### ISSUE TITLE
Really good feature that should be added.

### ISSUE THREAD

**Issue by Nigward8174** [UnamSanctam/UnamDownloader#40]
Would it be possible to make downloader use links from like rentry to download files everytime after pc was restarted? On that way just by changing link on rentry with miner's download link, we could keep everything updated with newer miner version

**Comment by UnamSanctam** [comment]
Yes it's possible, you'd just have to put a DownloadString inside the DownloadFile so that it gets the URL from the rentry first:
https://github.com/UnamSanctam/UnamDownloader/blob/3deb02a9e38dea2c11c11dcd502b3d6c4c42fe1e/UnamDownloader/Forms/Builder.cs#L184
It's a bit difficult to add into the builder though, not sure what it would look like.

**Comment by Nigward8174** [comment]
It would be nice to implement it into downloader itself because if miner crashes or something, that would make sure miner is working
